classes/class-p4-campaigns.php

```diff
@@ -297,13 +296,57 @@ public function validate_page_types( $selected_page_types ) : bool {
 		 * Load assets.
 		 */
 		public function enqueue_admin_assets() {
-			if ( ! is_admin() || strpos( get_current_screen()->taxonomy, $this->taxonomy ) === false ) {
+			if ( ! is_admin() || strpos( get_current_screen()->post_type, $this->post_type ) === false ) {
 				return;
 			}
 			wp_register_script( $this->taxonomy, get_template_directory_uri() . "/assets/js/$this->taxonomy.js", [ 'jquery' ], null, true );
 			wp_localize_script( $this->taxonomy, 'localizations', $this->localizations );
 			wp_enqueue_script( $this->taxonomy );
 			wp_enqueue_media();
 		}
+
```

Comment:
Please remove this whole function.
The campaign post type is registered in P4_Post_Campaign class